src/components/SignOut/SignOut.spec.tsx

```
@@ -2,35 +2,76 @@
+      name: JSON.stringify('Test User'),
+    };
+    return items[key] || null;
+  }),
+  setItem: vi.fn(),
+  removeItem: vi.fn(),
+  length: 0,
+  key: vi.fn(),
+};
+
+// Save the original localStorage
+const originalLocalStorage = global.localStorage;
```

ESLint

[error] 'global' is not defined.

(no-undef)